### PR TITLE
Calculate R(V)

### PR DESCRIPTION
**Pull request by mdecleir**
I added a function to calculate R(V), and a test function to test the calculations of A(V) and R(V).

### CODE REVIEW

**Comment by mdecleir** [comment]
The dev_test is failing with the following error:
`AttributeError: In accordance with NEP 32, the function fv was removed from NumPy version 1.20.  A replacement for this function is available in the numpy_financial library: https://pypi.org/project/numpy-financial`

**Comment by karllark** [comment]
No idea the origin of this error for the dev version.  I would ignore it for now.

File: measure_extinction/extdata.py [hunk 1]
```diff
@@ -423,15 +423,41 @@ def calc_AV(self, akav=0.112):
 
         # if no SpeX spectrum is available: compute A(V) from E(K-V)
         else:
-            dwaves = np.absolute(self.waves["BAND"] - 2.19 * u.micron)
-            sindxs = np.argsort(dwaves)
-            kindx = sindxs[0]
-            if dwaves[kindx] > 0.02 * u.micron:
-                warnings.warn("no K band measurement in E(lambda-V)", UserWarning)
+            kindx = np.where(self.waves["BAND"] == 2.19 * u.micron)[0][0]
```
Comment (karllark) [inline measure_extinction/extdata.py:426]: The previous code allowed some small variation in the K band central wavelength.  The idea here was that not all K bands were exactly the same (K vs Ks).   Is there a reason why this flexibility is not desired?  It was only allowing a variation of 0.02 micron.

File: measure_extinction/extdata.py [hunk 2]
```diff
@@ -423,15 +423,41 @@ def calc_AV(self, akav=0.112):
...
+        -------
+        Updates self.columns["RV"]
+        """
+        # obtain or calculate A(V)
+        if "AV" not in self.columns.keys():
+            self.calc_AV()
+        av = self.columns["AV"]
+
+        # obtain E(B-V)
+        bindx = np.where(self.waves["BAND"] == 0.438 * u.micron)[0][0]
```
Comment (karllark) [inline measure_extinction/extdata.py:452]: Again, exact wavelength.  Is that necessary or is something close to this wavelength ok?

File: measure_extinction/extdata.py (hunk 1 above)
Comment (mdecleir) [reply measure_extinction/extdata.py:426]: That is a good point. I thought the only purpose of the code was to find the K-band index. I did not realize the idea was to have some flexibility in the wavelength. However, the K-band wavelength has been hardcoded in `stardata.py` in the `get_poss_bands` function, so I don't see how there can be any variation in this wavelength as long as it's defined in that function.

Comment (karllark) [reply measure_extinction/extdata.py:426]: You are right, it isn't an issue now, but it could be in the future.  Just a thought.

Comment (mdecleir) [reply measure_extinction/extdata.py:426]: Ok, I'll add some more flexibility.

File: measure_extinction/extdata.py (hunk 2 above)
Comment (mdecleir) [reply measure_extinction/extdata.py]: I added some more flexibility now.

**Comment by mdecleir** [comment]
The tests are suddenly failing: 
`np.testing.assert_almost_equal(ext.columns["AV"], 2.5902545306388625)
E       AssertionError: 
E       Arrays are not almost equal to 7 decimals
E        ACTUAL: 2.602830960364401
E        DESIRED: 2.5902545306388625`

I don't understand this. Locally, the computed A(V)=2.5902545306388625. The test worked before the last commit, but the calculated A(V) should not change with this commit.

**Comment by mdecleir** [comment]
I realized what's going on now! It's a bit complicated but it has to do with the fact that you merged my other PR (with changes I made in another branch), and my local branch for this PR is not up to date with the master branch of the base... I thought it would be fine, because I was working on different parts of the code, but apparently one little change in the previous PR (cutting out a small part of the spectrum), changed the result of the extinction curve fit, and as a consequence A(V). Fixing it now!

**Comment by mdecleir** [comment]
Fixed! All checks are passing! Ready to be merged!

**Comment by karllark** [comment]
Great!

**Pull request merged**